java/code/webapp/WEB-INF/decorators/layout_error.jsp

```diff
@@ -79,6 +79,20 @@
             </div>
           </div>
         </div>
+        <rhn:require acl="user_authenticated()">
```

Comment:
Be aware that this page is a super sensitive one. It's used by the download endpoint only, and any change in here can hurt MLM server performance significantly. 
The change looks to be safe enough, but we need to keep that in mind.